meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -324,6 +325,10 @@ RESIN_CONFIGS[audit] = " \
     CONFIG_AUDITSYSCALL=n \
     "
 
+RESIN_CONFIGS_DEPS[governor] ?= " \
+    CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y \
+    "
```

Comment:
This will get used as being the last o one in the config file (which get's parsed by kconfig). This is the issue we are currently having with rpi - the default one is set by the BSP as powersave which makes all the people it when not knowing how to change the default (or not aware of it).